index.html

```diff
@@ -66,5 +60,4 @@
 </body>
```

Comment:
Inline JS such as like 53, is a no no. You want your functional code in your JS library, and not cluttering your HTML.